metadata-ingestion-modules/airflow-plugin/tests/integration/test_plugin.py

```diff
@@ -373,20 +377,26 @@ def test_airflow_plugin(
         print("Sleeping for a few seconds to let the plugin finish...")
         time.sleep(10)
 
-    _sanitize_output_file(airflow_instance.metadata_file)
-
-    check_golden_file(
-        pytestconfig=pytestconfig,
-        output_path=airflow_instance.metadata_file,
-        golden_path=golden_path,
-        ignore_paths=[
-            # TODO: If we switched to Git urls, maybe we could get this to work consistently.
-            r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['datahub_sql_parser_error'\]",
-            r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['openlineage_.*'\]",
-            r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['log_url'\]",
-            r"root\[\d+\]\['aspect'\]\['json'\]\['externalUrl'\]",
-        ],
-    )
+    """
+    we need to check that the golden file is missing / empty
+    when the dag_id is DAG_TO_SKIP_INGESTION
+    otherwise, this test doesn't actually do anything
```

Comment:
?

is this just a note to yourself to fix this tomorrow?

Reply:
no, in case if test skips then there will be no golden file gets generated, so basically we are not checking anything in case of skip excpet file is not present

Reply:
so we should have a `assert not Path(...).exists()`